packages/mailchimp/src/Adaptor.js

```diff
@@ -426,6 +428,48 @@ export const get = (path, query, callback) =>
 export const post = (path, body, query, callback) =>
   request('POST', path, { body, query }, callback);
 
+// fetch a bach result, extract and parse it, and return to state
+// TODO this takes the request_body_url explicitly, but could we just take the batch response?
+export const getBatchResult = (url) => {
```

Comment:
@mtuchi You'll need to give some thought as to exactly what you want this Operation to look like. Consider this a rough first draft.

We should add this logic to the mailchimp adaptor because it'll be useful later. We should not try to push any of this down to common - not yet, anyway.